### DISCUSSION TITLE
How to change where log files are stored?

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#210] [General]
How can I change where the log files are stored?

**Comment by rbeer** [comment]
Change the name / add a path to this line:
https://github.com/ThioJoe/YouTube-Spammer-Purge/blob/a859563ff1ac8e5751fac7a7e66d60ef97d299a8/YouTubeSpammerPurge.py#L2520

This ultimately uses Python's [open()](https://docs.python.org/3/library/functions.html#open) built-in, which is OS agnostic. So something like
```python
logFileName = "C:\logfiles\Spam_Log_" + datetime.now().strftime("%Y-%m-%d_%H-%M-%S" + ".rtf")
````
for Windows or
```python
logFileName = "/home/{YourUsername}/logfiles/Spam_Log_" + datetime.now().strftime("%Y-%m-%d_%H-%M-%S" + ".rtf")
```
for Linux/MacOS should be fine.

**Reply by kiatchun** [reply]
So I have to edit the code? I thought there would be a method to change it in the config file.

**Reply by rbeer** [reply]
Nope, sorry. The log file's path is hard coded.

**Comment by rbeer** [comment]
Added this as a config value in #226
